datafusion/core/tests/parquet/row_group_pruning.rs

```diff
@@ -141,7 +141,7 @@ async fn prune_timestamps_millis() {
     RowGroupPruningTest::new()
         .with_scenario(Scenario::Timestamps)
         .with_query(
-            "SELECT * FROM t where micros < to_timestamp_millis('2020-01-02 01:01:11Z')",
+            "SELECT * FROM t where millis < arrow_cast('2020-01-02T01:01:11', 'Timestamp(Millisecond, None)')",
```

Comment:
This test also seems to refer to the wrong column

```suggestion
            "SELECT * FROM t where micros < to_timestamp_millis('2020-01-02 01:01:11Z')",
```